sigopt/decorators.py

```diff
@@ -0,0 +1,11 @@
+# Copyright © 2024 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
```

Comment:
Why is this part of the orchestrate removal? Don't get me wrong, I think this is a useful idea, just not clear why it's part of this PR. Separate it out? 

Also, why only on the log-capture feature, not on other parts of the API?

Reply:
I wanted to improve vulture's ability to catch dead code so it could find more dangling orchestrate code bits. So I removed the vulture allowlist and added this bit so it wouldn't end up on the allowlist